website/blog/2018-12-21-state-of-react-native-community.md

```diff
@@ -0,0 +1,51 @@
+---
+title: The State the React Native Community in 2018
+author: Lorenzo Sciandra
+authorTitle: Core Maintainer & React Native Developer
+authorURL: https://github.com/kelset
+authorImageURL: https://avatars2.githubusercontent.com/u/16104054?s=460&v=4
+authorTwitter: kelset
+category: announcements
+---
+
+This year we made a large number of changes about how we develop and communicate about React Native; we hope that in two years from now we can look back and see that this shift was a turning point for React Native.
+
+A lot of people are excited about the rewrite of React Native's architecture, widely known as [Fabric](https://github.com/react-native-community/discussions-and-proposals/issues/4). Among other things, this will fix fundamental flaws in React Native's architecture and will set up React Native for success in the future together with [JSI and TurboModules](https://github.com/react-native-community/discussions-and-proposals/issues/40).
+
+The biggest shift initiated in 2018 was to empower the React Native Community: from the beginning Facebook encouraged developers from all around the world to participate in React Native's open source project. Since then, a number of core contributors emerged who handle, among other things, the release process.
+
+This year, these members took a few substantial steps towards making the whole community more able to shape the future of this project; two repositories have been created throughout the year precisely with this goal in mind:
```

Comment:
```suggestion
These members took a few substantial steps towards making the whole community more empowered to shape the future of this project. Two repositories have been created throughout the year precisely with this goal in mind:
```

- "This year" is the context of the entire post, no need to restate again
- Change "more able" to "more empowered"
- Remote semi-colon